src/utils/ExtensionLoader.js

```diff
@@ -401,24 +401,24 @@ define(function (require, exports, module) {
                     "HtmlEntityCodeHints",
                     "InlineColorEditor",
                     "JavaScriptQuickEdit",
-                    "JSLint",
-                    "LESSSupport",
+// XXXhumph: we aren't using these (yet), removed to save size
+//                   "JSLint",
+//                   "LESSSupport",
                     "QuickOpenCSS",
                     "QuickOpenHTML",
                     "QuickOpenJavaScript",
                     "QuickView",
-                    "RecentProjects",
+// XXXhumph: we don't yet support multiple projects, removing to save size
+//                  "RecentProjects",
                     "UrlCodeHints",
-                    "WebPlatformDocs",
+// XXXhumph: having this long term would be good, but removing for now to save size
+//                  "WebPlatformDocs",
```

Comment:
Note that the WebPlatformDocs extension only loads its docs dataset lazily, the first time its invoked -- so including that big css.json file in dist won't affect initial Brackets load time at all.